lib/ex_debug_toolbar/request.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ExDebugToolbar.Request do
+  alias ExDebugToolbar.Request.Event
+  alias ExDebugToolbar.Request
+
+  defstruct [
```

Comment:
not sure having a fixed struct is the best way. It hinders the ability to attach data, as everything we might track has to be pre-accepted in the request object